app/graphql/types/query_type.rb

```diff
@@ -43,5 +48,12 @@ def registration(id:)
     def course(id:)
       HourglassSchema.object_from_id(id, context)
     end
+
+    field :users, [UserType], null: false do
+      guard Guards::CURRENT_USER_ADMIN
```

Comment:
Subtle!  But correct -- in Bottlenose, an admin can only view all users when they're logged in as admin, and not currently  impersonating someone else